### ISSUE TITLE
Personal Portfolio Issues

### ISSUE THREAD

**Issue by skabeo** [kwesiObuobi/testing-with-drissa#1]
**Great job so far, but there are a few changes required.**

- The introduction should have a larger font size to make it easily readable.
- The nav links should be styled to have a good contrast ratio. Click [here](https://coolors.co/contrast-checker/112a46-acc8e5) to check contrast ratio.
- Flexbox should be used to display the project sections. This will make it adjust to different screen sizes.
- The introduction text should be centered.
